### DISCUSSION TITLE
How to debug Bot in Bot framework composer

### DISCUSSION THREAD

**Discussion by aparnaji** [microsoft/BotFramework-Composer#6243] [General]
How to debug Bot in Bot framework composer. One way is in emulator we can see the flow, apart from this is there a way we can debug in composer itself

**Comment by a-b-r-o-w-n** [comment]
@aparnaji in our next release, you will be able to interact with your bot via web chat (like emulator) directly from Composer. Beyond this, there is currently not any debugging support inside of Composer, but it is on our roadmap.
